app/services/imageService.js

```diff
@@ -0,0 +1,120 @@
+angular.module('portainer.services')
```

Comment:
Something has gone wrong during the merge of this file, you're not using the latest version of the `ImageService`.